bin/mustache_cli.ml

```diff
@@ -1,12 +1,27 @@
-let apply_mustache json_data template_data =
+module Mustache = struct
```

Comment:
Why do we need this module?

Reply:
This is a `Mustache` module where `With_locations` is the default. If I remove this declaration, `Mustache.parse_lx` and `Mustache.render` below will resolve to `Mustache.Without_locations`, so the errors will only receive dummy locations. Instead of defining this module, I could instead use `Mustache.With_locations.parse_lx` and `Mustache.With_locations.render` explicitly -- but I find the global declaration more convenient, as we never want to use the Without_locations definition in the binary.

(Which reminds me that I need to change the error-printing logic to avoid printing dummy locations. Will do right now.)

Reply:
I thought our plan was to make good locations the default, in which case this would only be temporary. In any case, it's fine regardless